content/_code-samples/non-fungible-token/py/burn-nft.py

```diff
@@ -0,0 +1,64 @@
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.models.transactions.nftoken_burn import NFTokenBurn
+from xrpl.models.requests import AccountNFTs
+from xrpl.clients import JsonRpcClient
+from xrpl.wallet import Wallet
+
+# Burn an NFT on the XRPL via a NFTokenBurn transaction
+# https://xrpl.org/nftokenburn.html#nftokenburn
+
+# For this code snippet to work, you need import an Account Seed with an issued NFT
+# You could use 'mint_nft.py' on the same folder to import an account with its owned minted NFT
+
+seed = ""
+
+if seed == "":
+    print("Please edit this code to update variable 'seed' to an account with a minted NFT to run this snippet. "
+          "You can get this by running 'mint-nft.py' and copying the printed seed.")
+else:
+    # Connect to a testnet node
+    print("Connecting to Testnet...")
+    JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+    client = JsonRpcClient(JSON_RPC_URL)
+
+    # Initialize wallet from seed
+    issuer_wallet = Wallet(seed=seed, sequence=0)
+    issuerAddr = issuer_wallet.classic_address
+
+    print(f"\nIssuer Account: {issuerAddr}")
+    print(f"          Seed: {issuer_wallet.seed}")
+
+    # Query the minter account for its NFTs
+    get_account_nfts = client.request(
+        AccountNFTs(account=issuerAddr)
+    )
+
+    nft_int = 1
+    print(f"\n - NFTs owned by {issuerAddr}:")
+    for nft in get_account_nfts.result['account_nfts']:
+        print(f"\n{nft_int}. NFToken metadata:"
+              f"\n    Issuer: {nft['Issuer']}"
+              f"\n    NFT ID: {nft['NFTokenID']}"
+              f"\n NFT Taxon: {nft['NFTokenTaxon']}")
+        nft_int += 1
+
+    print("\n/// --- ///")
+    print(f"Burning NFT {get_account_nfts.result['account_nfts'][0]['NFTokenID']}...")
```

Comment:
If the account has no NFTs, this throws an error currently

Reply:
Thanks for catching that :)